en/docs/monitoring/api-analytics/on-prem/opensearch-installation-guide.md

```
@@ -310,19 +304,20 @@ Create a parser configuration file (parsers.conf) with the following configurati
 
 2. Start Fluent Bit using the following command:
 
-    ```
-    fluent-bit -c <path_to_your_fluent-bit.conf>
-    ```
+   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)